SmellDetector/Services/DetectionService.cs

```diff
@@ -21,7 +21,8 @@ private void LoadDetectors()
             {
                 new LongMethodRuleEngine(),
                 new LongParameterListRuleEngine(),
-                new GodClassRuleEngine(),
+                new GodClassRuleEngine(47, 2, 0.33),
```

Comment:
Using the new design, this Service will only instantiate the ClassMetricsRuleEngine which will instantiate the rules that can have default values for their thresholds.